gapic-generator-cloud/templates/cloud/gem/readme.erb

```diff
@@ -0,0 +1,93 @@
+# Ruby Client for the <%= gem.title %> API
+
+<%= gem.summary %>
+
+<%= gem.description %>
+
+<%= gem.homepage %>
+
+## Installation
+
+```
+$ gem install <%= gem.name %>
+```
+
+## Before You Begin
+
+In order to use this library, you first need to go through the following steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+<%- if gem.free_tier? -%>
+1. This API has a free tier. You may not need to
+   [enable billing](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+   to get started.
+<%- else -%>
+1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+<%- end -%>
+<%- if gem.api_id -%>
+1. [Enable the API.](https://console.cloud.google.com/apis/library/<%= gem.api_id %>.googleapis.com)
+<%- end -%>
+1. {file:AUTHENTICATION.md Set up authentication.}
+
+## Quick Start
+
+<%- if gem.packages? -%>
+```ruby
+require "<%= gem.entrypoint_require %>"
+<%- service = gem.packages.first.services.first -%>
+<%- method = service&.methods.first -%>
+<%- if service && method -%>
+
+client = <%= service.create_client_call %>
+request = my_create_request
+response = client.<%= method.name %> request
+<%- end -%>
+```
+
+<%- end -%>
+<%- unless gem.library_documentation_url.to_s.empty? -%>
+View the [Client Library Documentation](<%= gem.library_documentation_url %>)
+for class and method documentation.
+
+<%- end -%>
+<%- if gem.product_documentation_url -%>
+See also the [Product Documentation](<%= gem.product_documentation_url %>)
+for general usage information.
+
+<%- end -%>
+## Enabling Logging
+
+To enable logging for this library, set the logger for the underlying [gRPC](https://github.com/grpc/grpc/tree/master/src/ruby) library.
+The logger that you set may be a Ruby stdlib [`Logger`](https://ruby-doc.org/stdlib/libdoc/logger/rdoc/Logger.html) as shown below,
+or a [`Google::Cloud::Logging::Logger`](https://googleapis.dev/ruby/google-cloud-logging/latest)
+that will write logs to [Stackdriver Logging](https://cloud.google.com/logging/). See [grpc/logconfig.rb](https://github.com/grpc/grpc/blob/master/src/ruby/lib/grpc/logconfig.rb)
```

Comment:
I think "Stackdriver Logging" is now "Cloud Logging"?

Reply:
Thanks! Fixed, and also fixed in the wrapper gem readmes.